docs/application/web/guides/machine-learning/overview.md

```diff
@@ -0,0 +1,24 @@
+# Machine Learning
+
+Machine Learning features help you to handle Neural Network Frameworks (NNFWs), like TensorFlow and TensorFlow-Lite.
+
+You can use the following machine learning features in your Web applications:
+
+- [Tensors Management](./tensors.md)
+
+  You can create and manage TenorsInfo and TensorsData objects, which can be used to exchange data with neural models.
+
+- [Single](./single.md)
+
+  Useful for a simple usage scenario of neural network models. It allows invoking a neural network model with a single instance of input data for the model directly.
```

Comment:
I'm not sure if 'You can...' isn't the official form for tizen-docs guides, annie-samsung mentioned that using third form is avoided in docs https://github.com/Samsung/tizen-docs/pull/1408#discussion_r589327370, however I don't know if this case counts for this limitation.

Reply:
Third form is commonly used in guides. It is avoided in docs. "You can" appears in guides 1223 times :)

Reply:
Done.